### PR TITLE
ci: fixing smoketest runners

### PR DESCRIPTION
**Pull request by CharlieMc0**
# Description

Fixes the smoke test workflow to work on public runners, removed a bunch of unused actions, and disabled the Discord token based release notification. We can do it much more safely with webhooks for a specific channel vs using a Discord bot token. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Include instructions and any relevant details so others can reproduce. 

- [ ] Tested CCTX in localnet
- [ ] Tested in development environment
- [ ] Go unit tests
- [ ] Go integration tests
- [ ] Tested via GitHub Actions 

# Checklist:

- [ ] I have added unit tests that prove my fix feature works

### CODE REVIEW

**Review by CryptoFewka — APPROVED** [review]
+1

File: .github/workflows/rc-release.yml [hunk 1]
```diff
@@ -82,43 +82,43 @@ jobs:
...
-          discord_channel_id: "${{ secrets.DISCORD_CHANNEL_ID }}"
-          discord_message: |
-            Hey <@&1122981184255840306>! A new version of the ZetaChain software has been released. 
-
-            Major Version Upgrade (e.g. v5.x.x to V6.x.x) must be completed through a governance proposal. 
-            We will submit a governance proposal in the next few days.
-            More specific information including block height will be shared as part of the governance proposal.
-
-            See the release notes for more details. https://github.com/zeta-chain/node/releases/tag/${{ env.BINARY_VERSION }}
+  # announce-release:
```
Comment (lumtis) [inline .github/workflows/rc-release.yml:85]: Can we maybe add a comment to explain the reason this is commented out?

**Review by lumtis — COMMENTED** [review]
This looks good to me, I'm just wondering about one thing.
```make zetanode``` step seems to take one additional minute to complete compared to before:
https://github.com/zeta-chain/node/actions/runs/7410644201/job/20163410705?pr=1436
https://github.com/zeta-chain/node/actions/runs/7414576117/job/20175927185?pr=1525
Any known reason for this?

File: .github/workflows/rc-release.yml (hunk 1 above)
Comment (CharlieMc0) [reply .github/workflows/rc-release.yml:85]: I'll just remove it entirely. The reason I am removing this is I don't want a Discord token that had high level of access to our Discord server to be accessible from a public repo.  I want to do better job keeping high value secrets away from public repos. Instead we can use GitHub's native Webhook support to post updates to a specific Discord channel. 

https://gist.github.com/jagrosh/5b1761213e33fc5b54ec7f6379034a22

I think we need a new GitHub-Releases discord channel and we can post updates from here, the contracts, Denis's template, etc.

**Comment by CharlieMc0** [comment]
> This looks good to me, I'm just wondering about one thing. `make zetanode` step seems to take one additional minute to complete compared to before: https://github.com/zeta-chain/node/actions/runs/7410644201/job/20163410705?pr=1436 https://github.com/zeta-chain/node/actions/runs/7414576117/job/20175927185?pr=1525 Any known reason for this?

Likely do to a change in runners. Our private runners make have had higher performance or a faster network connection but I not sure about that. 

We can do testing in another PR and compare the speed across different sized runners.

**Review by lumtis — APPROVED** [review]

**Pull request merged**